plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/ColumnIdentity.java

```diff
@@ -35,6 +35,7 @@
 import static io.trino.plugin.iceberg.ColumnIdentity.TypeCategory.MAP;
 import static io.trino.plugin.iceberg.ColumnIdentity.TypeCategory.PRIMITIVE;
 import static io.trino.plugin.iceberg.ColumnIdentity.TypeCategory.STRUCT;
+import static io.trino.plugin.iceberg.ColumnIdentity.TypeCategory.VARIANT;
```

Comment:
> Spike: Variant type read support for parquet

Please follow the commit message guideline. https://trino.io/development/process.html#pull-request-and-commit-guidelines-